linux_os/guide/system/auditing/auditd_configure_rules/directory_access_var_log_audit/ansible/shared.yml

```diff
@@ -0,0 +1,38 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+- name: Search /etc/audit/rules.d for audit rule entries
+  find:
+    paths: /etc/audit/rules.d
+    recurse: false
+    contains: ^.*dir=/var/log/audit/.*$
+    patterns: '*.rules'
+  register: find_var_log_audit
+
+- name: Use /etc/audit/rules.d/access-audit-trail.rules as the recipient for the rule
+  set_fact:
+    all_files:
+    - /etc/audit/rules.d/access-audit-trail.rules
+  when: find_var_log_audit.matched == 0
+
+- name: Use matched file as the recipient for the rule
+  set_fact:
+    all_files:
+    - '{{ find_var_log_audit.files | map(attribute=''path'') | list | first }}'
+  when: find_var_log_audit.matched &gt; 0
+
+- name: Inserts/replaces the /var/log/audit/ rule in rules.d
+  lineinfile:
+    path: '{{ all_files[0] }}'
+    line: -a always,exit -F dir=/var/log/audit/ -F perm=r -F auid&gt;=1000 -F auid!=unset
+      -F key=access-audit-trail
+    create: true
+
+- name: Inserts/replaces the /var/log/audit/ rule in audit.rules
+  lineinfile:
+    path: /etc/audit/audit.rules
+    line: -a always,exit -F dir=/var/log/audit/ -F perm=r -F auid&gt;=1000 -F auid!=unset
```

Comment:
The Ansible templates use ``-F auid >=1000``, e.g. 

`````
grep -rin "auid"  template_ANSIBLE_*                                     131  19:15:49 
template_ANSIBLE_audit_rules_dac_modification:40:    line: "-a always,exit -F arch=b32 -S {{{ ATTR }}} -F auid>={{{ auid }}} -F auid!=unset -F key=perm_mod"
template_ANSIBLE_audit_rules_dac_modification:46:    line: "-a always,exit -F arch=b64 -S {{{ ATTR }}} -F auid>={{{ auid }}} -F auid!=unset -F key=perm_mod"
template_ANSIBLE_audit_rules_dac_modification:54:    line: "-a always,exit -F arch=b32 -S {{{ ATTR }}} -F auid>={{{ auid }}} -F auid!=unset -F key=perm_mod"
template_ANSIBLE_audit_rules_dac_modification:60:    line: "-a always,exit -F arch=b64 -S {{{ ATTR }}} -F auid>={{{ auid }}} -F auid!=unset -F key=perm_mod"
template_ANSIBLE_audit_rules_file_deletion_events:40:    line: "-a always,exit -F arch=b32 -S {{{ NAME }}} -F auid>={{{ auid }}} -F auid!=unset -F key=delete"
template_ANSIBLE_audit_rules_file_deletion_events:46:    line: "-a always,exit -F arch=b64 -S {{{ NAME }}} -F auid>={{{ auid }}} -F auid!=unset -F key=delete"
template_ANSIBLE_audit_rules_file_deletion_events:54:    line: "-a always,exit -F arch=b32 -S {{{ NAME }}} -F auid>={{{ auid }}} -F auid!=unset -F key=delete"
template_ANSIBLE_audit_rules_file_deletion_events:60:    line: "-a always,exit -F arch=b64 -S {{{ NAME }}} -F auid>={{{ auid }}} -F auid!=unset -F key=delete"
template_ANSIBLE_audit_rules_path_syscall:43:    - "-a always,exit -F arch=b32 -S {{{ SYSCALL }}} -F a2&03 -F path={{{ PATH }}} -F auid>={{{ auid }}} -F auid!=unset -F key=modify"
template_ANSIBLE_audit_rules_path_syscall:51:    - "-a always,exit -F arch=b64 -S {{{ SYSCALL }}} -F a2&03 -F path={{{ PATH }}} -F auid>={{{ auid }}} -F auid!=unset -F key=modify"
template_ANSIBLE_audit_rules_path_syscall:62:    - "-a always,exit -F arch=b32 -S {{{ SYSCALL }}} -F a2&03 -F path={{{ PATH }}} -F auid>={{{ auid }}} -F auid!=unset -F key=modify"
template_ANSIBLE_audit_rules_path_syscall:71:    - "-a always,exit -F arch=b64 -S {{{ SYSCALL }}} -F a2&03 -F path={{{ PATH }}} -F auid>={{{ auid }}} -F auid!=unset -F key=modify"
`````

Which should be used?

Reply:
@shawndwells fixed